### DISCUSSION TITLE
这个能设置gpu，cpu的开关吗，我看cpu跑会把cpu资源跑没

### DISCUSSION THREAD

**Discussion by xuboot** [hiroi-sora/Umi-OCR#649] [Q&A]
这个能设置gpu，cpu的开关吗，我看cpu跑会把cpu资源跑没

**Comment by hiroi-sora** [comment]
暂时不支持GPU运行，我们前期测试的时候认为这对驱动和环境的要求比较高，兼容性比较差，可能需要用户有一定动手能力来配置系统，所以没有正式上线此功能。

👉 https://github.com/qwedc001/RapidOCR-json-gpu

我们未来会寻找更易使用的GPU任务组件并适配给Umi。
